ebean-core/src/main/java/io/ebeaninternal/server/core/OrmQueryRequest.java

```diff
@@ -469,7 +486,10 @@ public BeanPropertyAssocMany<?> manyPropertyForOrderBy() {
    * query plan for this query exists.
    */
   public CQueryPlan queryPlan() {
-    return beanDescriptor.queryPlan(queryPlanKey);
+    if (queryPlan == null) {
+      queryPlan = beanDescriptor.queryPlan(queryPlanKey);
```

Comment:
here we cache the current query plan, that made the "dependentTables" refactoring possible